File: sample_codes/python/auth_api/get-v2-oauth2-verify_token.py
import requests, json

end_point = 'https://public-api.hakaru.ai'
path = '/v2/oauth2/verify_token'
url = end_point + path

access_token = 'ACCESS_TOKEN'

headers = { 'Authorization': 'Bearer ' + access_token }

try:
    response = requests.get(url, headers=headers)
    response.raise_for_status()

    print('HTTP Status Code:', response.status_code)
except requests.exceptions.RequestException as err:
    print(err)
finally:
    print('Response Body:', json.dumps(response.json(), indent=2))

File: sample_codes/php/auth_api/get-v2-oauth2-verify_token.php
<?php
$endPoint = "https://public-api.hakaru.ai";
$path = "/v2/oauth2/verify_token";
$url = $endPoint . $path;

$accessToken = "ACCESS_TOKEN";

$ch = curl_init($url);
curl_setopt($ch, CURLOPT_HTTPHEADER, array('Authorization: Bearer ' . $accessToken));
curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);

$strResult = curl_exec($ch);
$httpcode = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);

curl_close($ch);

$resultObj = json_decode($strResult);
$result = json_encode($resultObj, JSON_UNESCAPED_UNICODE | JSON_PRETTY_PRINT);

print_r("HTTP Status Code: " . $httpcode . "\n");
print_r("Response Body: " . $result . "\n");

?>